src/data/uma-title.csv

```diff
@@ -59,25 +59,25 @@
 "伝説の歌姫","Legendary Diva"
 "スキルマスター","Skill Mastery"
 "技のデパート","Various Technique"
-"日本の総大将",""
+"日本の総大将","General of Japanese"
 "異次元の逃亡者","Dimensional Runaway"
 "怪鳥","Eerie Bird"
 "スーパーカー","Supercar"
 "変幻自在","Shape Shifter"
 "驀進王","Bakushin・O"
 "高速ステイヤー","High Speed Stayer"
 "不死鳥","Phoenix"
-"常識破りの女帝",""
+"常識破りの女帝","The Empress of Common Sense"
 "ミスパーフェクト","Miss Perfect"
 "帝王","Teio・Emperor"
 "破天荒","Unheard"
 "皇帝","Kotei・Emperor"
 "笑門来福",""
-"愛しき名脇役",""
+"愛しき名脇役","Lovely Supporting Role"
```

Comment:
Beloved Supporting Actor

(Although the japanese includes both "Beloved" and "Excellent", "Beloved Excellent Supporting Actor" is very weird in english, so restricting it to Beloved seems fine to me)